include/NovelRT/Transform.h

```diff
@@ -16,42 +19,79 @@ namespace NovelRT {
     bool _active;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
+     * @param position The position.
+     * @param rotation The rotation.
+     * @param scale The scale.
```

Comment:
Maybe describe these a little more? i.e. `scale` represents the size, for example.

Reply:
I'll try